src/test/regress/citus_tests/test/test_maintenancedeamon.py

```diff
@@ -0,0 +1,26 @@
+# This test checks that once citus.main_db is set and the
+# server is restarted. A Citus Maintenance Daemon for the main_db
+# is launched. This should happen even if there is no query run
+# in main_db yet.
+def test_set_maindb(coord):
+    with coord.cur() as cur1:
+        cur1.execute("DROP DATABASE IF EXISTS mymaindb;")
+        cur1.execute("CREATE DATABASE mymaindb;")
```

Comment:
Let's track the databases we create so we can clean them up automatically, just like we do for schemas. See `create_schema`, `cleanup_schemas` and `cleanup_test_leftovers` in `src/test/regress/citus_tests/common.py`